src/main/java/org/edumips64/core/CacheSimulator.java

```diff
@@ -41,248 +47,125 @@ public CacheMemory getL1InstructionCache() {
         return L1InstructionCache;
     }
 
+    // Re-export CacheType for backward compatibility
```

Comment:
Can we avoid to have this enum in 2 different places? If it's the cleanest way, feel free to remove CacheType.java.